packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/lib/base/common_services/count_down_service.dart

```diff
@@ -0,0 +1,17 @@
+{{> licence.dart }}
+
+import 'dart:async';
+
+import 'package:rxdart/rxdart.dart';
+
+class CountDownService {
+  static const maxTime = 50;
```

Comment:
Do we really need those static properties publicly avaialbe ?

Reply:
Not needed. Removed.